views/index.html

```diff
@@ -11,7 +11,7 @@ <h2 id="message-title">Thank you</h2>
 </span>
 
 <section id="join" class="u-full-width header">
-    <h1><b>Join</b> dvcoders</h1>
+    <h1><b>Join</b> Dvcoders</h1>
```

Comment:
I think we should keep it as `dvcoders`, I remember we had this discussion in the club before. The founding president, Daniel Awai, said it is dvcoders, without capitalized D.

cc @thebearjew thoughts?